src/components/form/checkbox/_checkbox.scss

```diff
@@ -3,6 +3,10 @@
 
   &:not(.euiCheckbox--noLabel) .euiCheckbox__input {
     @include euiScreenReaderOnly;
+    // Because this input is still focusable, it causes the screen-reader outline
+    // to extend to the position of the screen-reader element.
+    // So we must position it against it's own bounds.
+    left: 0;
```

Comment:
These parts will be dependent  on your thoughts on https://github.com/elastic/eui/pull/5152#discussion_r704556686

Reply:
This ends up looking a bit weird on Firefox+VO, unfortunately:

<img width="199" alt="" src="https://user-images.githubusercontent.com/549407/132560095-13476fd3-2eb7-4b48-a814-6b65952cb637.png">

Reply:
Sorry, was doing a bit more testing cross-browser and didn't complete my thought 😅 TBH, I'm fine with shipping with this still, esp. since Safari+VO has way more usage than FF+VO. It's probably impossible to get everything exactly right in all browsers+screen readers, and this is still perfectly usable in FF. Also Chrome+VO looks the same as before which is great 👍